README.md

```diff
@@ -1,30 +1,34 @@
 # @artsy/palette [![CircleCI](https://circleci.com/gh/artsy/palette.svg?style=shield)](https://circleci.com/gh/artsy/palette) [![npm version](https://badge.fury.io/js/%40artsy%2Fpalette.svg)](https://www.npmjs.com/package/@artsy/palette) [![Netlify Status](https://api.netlify.com/api/v1/badges/beb9e8d7-10cc-4a2e-99bb-0d4c6f46db82/deploy-status)](https://app.netlify.com/sites/artsy-palette/deploys)
 
-Artsy's Design System
+# Artsy's Design System
 
-## Meta
+Point People: [@dzucconi](https://github.com/dzucconi), [@damassi](https://github.com/damassi)
 
-- Storybook: [https://palette-storybook.artsy.net](https://palette-storybook.artsy.net)
-- Docs: [https://palette.artsy.net](https://palette.artsy.net)
-- Docs Github Repo: [https://github.com/artsy/palette-docs](https://github.com/artsy/palette-docs)
-- Point People: [@dzucconi](https://github.com/dzucconi), [@damassi](https://github.com/damassi)
+### Storybook 
+[https://palette-storybook.artsy.net](https://palette-storybook.artsy.net) 
+The latest design system and components are available here 🙌🏻 We have a new repo for icons and prefer to use that one ⬇️ 
+Icons will be removed from palette-storybook.artsy.net eventually
+
+### Icons 
+[https://icons.artsy.net/](https://icons.artsy.net/) use this for icons
+
+### Other links
+Docs [https://palette.artsy.net](https://palette.artsy.net) To be depracated. It contains old design system and components. Docs will eventually move to the palette-storybook.artsy.net
+Docs Github Repo: [https://github.com/artsy/palette-docs](https://github.com/artsy/palette-docs)
 
 ## What is Palette?
 
-Palette is a collection of primitive, product-agnostic elements that help encapsulate Artsy's look and feel at base level. This project is intended to be used
-across our digital product portfolio.
+Palette is a collection of primitive, product-agnostic elements that help encapsulate Artsy's look and feel at base level. This project is intended to be used across our digital product portfolio.
 
 ## Does my component belong in Palette?
 
-If the component applies to Artsy as a brand and can/will be used across multiple digital products, then Palette is a great place for it. If it's highly product
-specific then it's best to leave the component where it's used. We can always move things later!
+If the component applies to Artsy as a brand and can/will be used across multiple digital products, then Palette is a great place for it. If it's highly product specific then it's best to leave the component where it's used. We can always move things later.
 
 If the above guidance still doesn't give you a good sense of what to do, please join the bi-weekly design systems sync.
```

Comment:
@damassi is the bi-weekly design systems sync still a thing? I would love more info about it yes, otherwise we could remove it.

Reply:
You can remove this. This must pre-date me, even.